Index: elizavetadoronina/pom.xml
===================================================================
diff --git a/elizavetadoronina/pom.xml b/elizavetadoronina/pom.xml
deleted file mode 100644
--- a/elizavetadoronina/pom.xml	(revision 1692c8bb91af8efd7fdb989181d5a7558a7a0e8c)
+++ /dev/null	(revision 1692c8bb91af8efd7fdb989181d5a7558a7a0e8c)
@@ -1,18 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.mipt</groupId>
-  <artifactId>elizavetadoronina</artifactId>
-  <packaging>jar</packaging>
-  <version>1.0-SNAPSHOT</version>
-  <name>elizavetadoronina</name>
-  <url>http://maven.apache.org</url>
-  <dependencies>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>3.8.1</version>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-</project>
Index: elizavetadoronina/src/main/java/com/mipt/App.java
===================================================================
diff --git a/elizavetadoronina/src/main/java/com/mipt/App.java b/elizavetadoronina/src/main/java/com/mipt/App.java
deleted file mode 100644
--- a/elizavetadoronina/src/main/java/com/mipt/App.java	(revision 1692c8bb91af8efd7fdb989181d5a7558a7a0e8c)
+++ /dev/null	(revision 1692c8bb91af8efd7fdb989181d5a7558a7a0e8c)
@@ -1,13 +0,0 @@
-package com.mipt;
-
-/**
- * Hello world!
- *
- */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
-    }
-}
Index: elizavetadoronina/src/test/java/com/mipt/AppTest.java
===================================================================
diff --git a/elizavetadoronina/src/test/java/com/mipt/AppTest.java b/elizavetadoronina/src/test/java/com/mipt/AppTest.java
deleted file mode 100644
--- a/elizavetadoronina/src/test/java/com/mipt/AppTest.java	(revision 1692c8bb91af8efd7fdb989181d5a7558a7a0e8c)
+++ /dev/null	(revision 1692c8bb91af8efd7fdb989181d5a7558a7a0e8c)
@@ -1,38 +0,0 @@
-package com.mipt;
-
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
-
-/**
- * Unit test for simple App.
- */
-public class AppTest 
-    extends TestCase
-{
-    /**
-     * Create the test case
-     *
-     * @param testName name of the test case
-     */
-    public AppTest( String testName )
-    {
-        super( testName );
-    }
-
-    /**
-     * @return the suite of tests being tested
-     */
-    public static Test suite()
-    {
-        return new TestSuite( AppTest.class );
-    }
-
-    /**
-     * Rigourous Test :-)
-     */
-    public void testApp()
-    {
-        assertTrue( true );
-    }
-}
